shared/actions/chat/index.js

```diff
@@ -853,10 +809,7 @@ function * _badgeAppForChat (action: Constants.BadgeAppForChat): SagaGenerator<a
     conversationsWithKeys[Constants.conversationIDToKey(conv.get('convID'))] = conv.get('UnreadMessages')
   })
   const conversationUnreadCounts = Map(conversationsWithKeys)
-  yield put({
-    payload: conversationUnreadCounts,
```

Comment:
i think its better to have all payloads be objects and not have these cases where the payload is a naked value